src/definition.rs

```diff
@@ -114,13 +117,25 @@ impl TemplateDefinition {
                     continue;
                 }
                 Value::String(s) => {
-                    let variables = has_template_variables(&s, &vals);
-                    let default_value = match variables {
-                        Some(variables) => replace_with_previous_responses(
-                            variables, &vals, &s),
-                        None => s.clone(),
+                    let contains_template = has_template_variables(&s);
```

Comment:
you can just check `if s.contains("{{") && s.contains("}}")` rather than using regex